.changeset/beige-forks-sit.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ts-rest/core': minor
+---
+
+fix client not validating response
```

Comment:
We capitalize these because they are part of the changelog

Reply:
Done 0d0af4f